package/MDAnalysis/topology/PDBParser.py

```diff
@@ -37,62 +47,172 @@
    :inherited-members:
 
 """
-from __future__ import absolute_import
+from __future__ import absolute_import, print_function
 
-try:
-    # BioPython is overkill but potentially extensible (altLoc etc)
-    import Bio.PDB
-except ImportError:
-    raise ImportError("Bio.PDB from biopython not found."
-                      "Required for PDB topology parser.")
+import numpy as np
+import warnings
 
-from .base import TopologyReader
 from ..core.AtomGroup import Atom
-from ..coordinates.pdb.extensions import get_structure
-from .core import guess_atom_type, guess_atom_mass, guess_atom_charge
+from .core import get_atom_mass, guess_atom_element
+from ..lib.util import openany
+from .base import TopologyReader
 
 
 class PDBParser(TopologyReader):
-    """Read minimum topology information from a PDB file."""
+    """Parser that obtains a list of atoms from a standard PDB file.
+
+    See Also
+    --------
+    :class:`MDAnalysis.coordinates.PDB.PDBReader`
+
+    .. versionadded:: 0.8
```

Comment:
I think we introduced class-based topology parsers in 0.8. (But PDB-parsing functionality was there pretty much right from 0.4 or thereabout).

I think it's ok to leave it there. But do add

```rest
.. versionchanged: 0.15.0
   Does not use the BioPython PDBParser any more.
```